core/src/executor/evaluate/function.rs

```diff
@@ -50,6 +50,33 @@ macro_rules! eval_to_float {
 
 // --- text ---
 
+pub fn concat(name: String, exprs: Vec<Evaluated<'_>>) -> Result<Value> {
+    let mut s = String::from("");
+
+    for expr in exprs {
+        match expr.try_into()? {
+            Value::Bool(value) => s.push_str(&value.to_string()),
+            Value::Date(value) => s.push_str(&value.to_string()),
+            Value::Decimal(value) => s.push_str(&value.to_string()),
+            Value::I8(value) => s.push_str(&value.to_string()),
+            Value::I64(value) => s.push_str(&value.to_string()),
+            Value::F64(value) => s.push_str(&value.to_string()),
+            Value::Str(value) => s.push_str(&value),
+            Value::Timestamp(value) => s.push_str(&value.to_string()),
+            //Value::Interval(value) => s.push_str(&value.to_string()),
+            Value::Uuid(value) => s.push_str(&value.to_string()),
+            //Value::Map(value) => s.push_str(value.to_string()),
+            //Value::List(value) => s.push_str(value.to_string()),
+            Value::Null => {} // ignore null values
+            _ => {
+                return Err(EvaluateError::FunctionRequiresStringValue(name).into());
+            }
+        }
+    }
```

Comment:
We can still use `Value::concat`.
`Value::concat` handles the operation between other values and `Null` to make `Null`.
However, simply we can filtering out `Null` and use `Value::concat`

Reply:
ok.   I changed the code to use Value::concat.